docs/sdk/server-side-sdks/php/php-install.md

```diff
@@ -38,4 +38,54 @@ Download the files and include `autoload.php`:
 require_once('/path/to/DevCycle/vendor/autoload.php');
 ```
 
+## Local Bucketing Proxy
+
+Due to complexities with the PHP application lifecycle and state management, local bucketing functionality requires a different approach compared to the other server SDKS.
+
+To access this functionality in PHP, DevCycle provides a **proxy** process that can run alongside your PHP application or on a separate host in your environment. This proxy mimics the Cloud Bucketing API but provides for higher speed variable evaluations due to the reduction of network latency and config caching; all powered by DevCycle's high performance Go Server SDK.
+
+The local bucketing proxy can be downloaded as a binary or package from here: https://github.com/DevCycleHQ/local-bucketing-proxy/releases
+
+#### Proxy Configuration
+
+Once the proxy is downloaded and installed, it can be configured via either environment variables or command line arguments. See the [README](https://github.com/DevCycleHQ/local-bucketing-proxy#options) for the configuration options.
+
+At a minimum, you will need set the `DVC_LB_PROXY_SDK_KEY` environment variable to activate the proxy. This should be set to your SDK key for your PHP application.
+The rest of the values can be left unchanged for basic operation. This will default to starting an HTTP server on port 8080.
+
+
+#### Running the Proxy
+
+The proxy can be started by just executing the binary:
+
+```bash
+
+$ local-bucketing-proxy
```

Comment:
that should really be named `dvc-local-bucketing-proxy` or something devcycle specific.

Reply:
probably actually: `devcycle-local-bucketing-proxy`

Reply:
yeah, that is probably a good idea

Reply:
https://github.com/DevCycleHQ/local-bucketing-proxy/pull/1